app/Http/Controllers/ScholarPrePhdSeminarController.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\PrePhdSeminar;
+use App\Models\Scholar;
+use App\Models\ScholarAppeal;
+use App\Types\RequestStatus;
+use App\Types\ScholarAppealTypes;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+
+class ScholarPrePhdSeminarController extends Controller
+{
+    public function request(Scholar $scholar)
+    {
+        $this->authorize('apply', [PrePhdSeminar::class, $scholar]);
+
+        return view('research.scholars.pre_phd_form', [
+            'scholar' => $scholar,
+        ]);
+    }
+
+    public function show(Scholar $scholar, PrePhdSeminar $appeal)
+    {
+        $this->authorize('view', [PrePhdSeminar::class, $scholar, $appeal]);
+
+        return view('research.scholars.pre_phd_form', [
+            'scholar' => $scholar,
+        ]);
+    }
+
+    public function apply(Scholar $scholar)
+    {
+        $this->authorize('apply', [PrePhdSeminar::class, $scholar]);
+
+        $scholar->prePhdSeminar()->create([
+            'status' => RequestStatus::APPLIED,
+        ]);
+
+        flash('Request for Pre-PhD Seminar applied successfully!')->success();
+
+        return redirect(route('scholars.profile'));
+    }
+
+    public function forward(Request $request, Scholar $scholar, PrePhdSeminar $appeal)
+    {
+        $this->authorize('forward', [PrePhdSeminar::class, $scholar, $appeal]);
```

Comment:
You can do the same thing as suggested above and other places where er applicable.